addon/ng2/blueprints/enum/index.js

```diff
@@ -0,0 +1,48 @@
+const stringUtils = require('ember-cli-string-utils');
+var dynamicPathParser = require('../../utilities/dynamic-path-parser');
+var addBarrelRegistration = require('../../utilities/barrel-management');
+
+module.exports = {
+  description: '',
+
+  normalizeEntityName: function (entityName) {
+    var parsedPath = dynamicPathParser(this.project, entityName);
+
+    this.dynamicPath = parsedPath;
+    return parsedPath.name;
+  },
+
+  locals: function (options) {
+    var classType = options.args[2];
+
+    this.fileName = stringUtils.dasherize(options.entity.name);
+    if (classType) {
+      this.fileName += '.' + classType;
+    }
```

Comment:
Please remove references to `classType` this is a holdover from the class blueprint which is not needed for enums